index.md

```diff
@@ -1,15 +1,49 @@
 ---
 layout: default
-title: Secret Hitler Strategy
+title: The Unofficial Secret Hitler Strategy Guide
 ---
 
-# Secret Hitler Strategy
+# The Unofficial Secret Hitler Strategy Guide
 
-Strategies learned from 300+ games
+[Secret Hitler](http://secrethitler.com/) is a social deduction game for 5–10 people about finding and stopping the Secret Hitler.
```

Comment:
Hemingway Editor suggests this is a bit hard to read. Maybe we could reword to something like:

Secret Hitler is a social deduction game for 5–10 people. The aim is to find and stop Secret Hitler.